src/modules/household/components/elements/AddToHouseholdButton.tsx

```diff
@@ -84,7 +85,7 @@ const AddToHouseholdButton = ({
         }
       },
     }),
-    [project.id, clientId, householdId, cocCount, onSuccess, household]
+    [project.id, clientId, cocCount, onSuccess, household]
```

Comment:
This is probably ok, just noting that now it will rerun when `household` changes as opposed to only when the householdId changes. Not sure if there is any impact in practice, but there might be. Could just watch `household?.id` instead